### PR TITLE
tests for demoqa/text-box

### PR DESCRIPTION
**Pull request by Greta-mart**
DemoQA site
Text-box page

### CODE REVIEW

File: tests/threc/HW4_demoqa/test_textbox.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import constants
+
+
+def test_textbox(driver):
+    driver.find_element(By.ID, 'userName').send_keys(constants.FIRST_NAME)
+    driver.find_element(By.ID, 'userEmail').send_keys(constants.USER_EMAIL)
+    driver.find_element(By.ID, 'currentAddress').send_keys(constants.CURRENT_ADDRESS)
+    driver.find_element(By.ID, 'permanentAddress').send_keys(constants.PERMANENT_ADDRESS)
+
+    driver.implicitly_wait(3)
```
Comment (fkudinov-mn) [inline tests/threc/HW4_demoqa/test_textbox.py:13]: move to driver fixture

File: tests/threc/HW4_demoqa/test_textbox.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    first_name = driver.find_element(By.ID, 'name').text
+    just_first_name = first_name.lstrip('Name:')
+    assert just_first_name == constants.FIRST_NAME
+
+    user_email = driver.find_element(By.ID, 'email').text
+    just_email = user_email.lstrip('Email:')
+    assert just_email in constants.USER_EMAIL
+
+    permanent_address = driver.find_element(By.ID, 'permanentAddress').text
+    just_permanent_address = permanent_address.lstrip('Permananet Address :')
```
Comment (fkudinov-mn) [inline tests/threc/HW4_demoqa/test_textbox.py:32]: do not use lstrip to remove prefix

Comment (fkudinov-mn) [reply tests/threc/HW4_demoqa/test_textbox.py:32]: image

**Review by kudinov-fedor — APPROVED** [review]

File: tests/threc/HW4_demoqa/conftest.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import pytest
+from selenium.webdriver.common.by import By
+
+import constants
+
+from selenium.webdriver import Chrome
+
+
+@pytest.fixture(autouse=True)
```
Comment (kudinov-fedor) [inline tests/threc/HW4_demoqa/conftest.py:9]: remove autouse

File: tests/threc/HW4_demoqa/test_textbox.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    first_name = driver.find_element(By.ID, 'name').text
+    just_first_name = first_name.removeprefix('Name:')
+    assert just_first_name == constants.FIRST_NAME
+
+    user_email = driver.find_element(By.ID, 'email').text
+    just_email = user_email.removeprefix('Email:')
+    assert just_email in constants.USER_EMAIL
+
+    permanent_address = driver.find_element(By.ID, 'permanentAddress').text
+    just_permanent_address = permanent_address.removeprefix('Permananet Address :')
```
Comment (kudinov-fedor) [inline tests/threc/HW4_demoqa/test_textbox.py:30]: take into account 
removeprefix works for python >= 3.9

**Review by kudinov-fedor — APPROVED** [review]

**Pull request merged**